README.md

```diff
@@ -87,15 +87,15 @@ if it does not work for your use-case.
 ### Install dependencies (Windows)
 
 We provide prebuilt dependencies to build the project. Download them
-with the [scripts\install_deps.bat](scripts\install_deps.bat) script.
+with the [scripts\install_deps.bat](scripts/install_deps.bat) script.
```

Comment:
Bring back the \

Reply:
Sure, but the link will lead to 404 page on github, as its doing now. Is it really intended?

Reply:
Yes, you are right. Sorry.

Reply:
No problem. Glad I could help.